android/sdk/src/main/java/com/taobao/weex/common/WXErrorCode.java

```diff
@@ -210,6 +210,7 @@
    */
   WX_RENDER_ERR_JS_CREATE_INSTANCE("-9600", "white screen cause create instance failed,check js stack ->",ErrorType.RENDER_ERROR,ErrorGroup.JS),
   WX_RENDER_ERR_JS_CREATE_INSTANCE_CONTEXT("-9601", "white screen cause create instanceContext failed,check js stack ->",ErrorType.RENDER_ERROR,ErrorGroup.JS),
+  WX_RENDER_ERR_EAGLE_RENDER("-9630", "eagle render err, stack ->",ErrorType.RENDER_ERROR,ErrorGroup.JS),
```

Comment:
1. Rename it to `WX_DEGRADE_EAGLE_RENDER_ERROR` and move it to line 265.
1. `ErrorType.DEGRAD_ERROR` would be more appropriate as it will cause degrade.